series/series_test.go

```diff
@@ -1478,6 +1478,16 @@ func TestSeries_Set(t *testing.T) {
 			Ints([]string{"1", "2", "3"}),
 			"[A 1 2 K 3]",
 		},
+		{
+			Strings([]string{"A", "B", "C"}),
+			[]bool{true, true, true},
+			New([]interface{}{
+				"A",
+				stringElement{e: "", nan: true},
```

Comment:
Is there other type such as `intElement` ? Will it work for such case?

Reply:
for `intElement` it is working expectedly, for float is not I'll update the floatElement